src/utils.lisp

```diff
@@ -35,29 +35,27 @@
         *dispatch-table*))
 
 (defun alist-keys (alist)
-  (loop for (key . value) in alist
-        collect key))
+  (mapcar #'car alist))
 
 (defun color-for-board (board)
   (or (cassoc :color (cassoc board *boards* :test #'string=))
       "#00ff00"))
 
 (defun trim-whitespace (str)
-  (string-trim '(#\Space #\Newline #\Backspace #\Tab #\Linefeed #\Page #\Return #\Rubout)
+  (string-trim '(#\Space #\Newline #\Backspace #\Tab
+                 #\Linefeed #\Page #\Return #\Rubout)
                str))
 
 (defun instance-has-backgrounds-p ()
-  (> (length  (loop for (key . value)
-                      in *boards*
-                    when (cassoc :background value)
-                      collect (cassoc :background value)))
-     0))
```

Comment:
In Common Lisp, anything that is not the empty list NIL or '() is considered true.
No need to check length against 0.